docs/docs/adrs/adr-005-cryptographic-equivocation-verification.md

```diff
@@ -8,6 +8,7 @@ title: Cryptographic verification of equivocation evidence
 * 5/1/2023: First draft
 * 7/23/2023: Add light client attacks handling
 * 9/6/2023: Add double signing attacks handling
+* 11/3/2023: Update limitations to handle amnesia attacks
```

Comment:
```suggestion
* 11/3/2023: Update limitations to clarify amnesia attacks are ignored
```
This might be better at expressing what was changed in the adr? I didn't get the original changelog line